pkg/ui/src/views/app/components/layoutSidebar/index.tsx

```diff
@@ -88,13 +89,17 @@ function LoginIndicator({ loginState, handleLogout }: LoginIndicatorProps) {
 
   return (
     <li className="login-indicator">
-        <div
-          className="login-indicator__initial"
-          title={`Signed in as ${user}`}
-        >
-          { user[0] }
+      <Link to={LOGOUT_PAGE} onClick={handleLogout}>
+        <div>
+          <div
+            className="login-indicator__initial"
+            title={`Signed in as ${user}`}
```

Comment:
as long as you're in here, can you update this to use `TooltipWrapper` please?

Reply:
More CSS struggles when I insert the `ToolTipWrapper` :P 
![image](https://user-images.githubusercontent.com/7341/46638244-f3db4300-cb2d-11e8-90a5-fe0f7fae5aeb.png)